api/git/azure.go

```diff
@@ -247,7 +247,7 @@ func (a *azureDownloader) buildDownloadUrl(config *azureOptions, referenceName s
 }
 
 func (a *azureDownloader) buildRefsUrl(config *azureOptions, referenceName string) (string, error) {
-	rawUrl := fmt.Sprintf("%s/%s/%s/_apis/git/repositories/%s/refs",
+	rawUrl := fmt.Sprintf("%s/%s/%s/_apis/git/repositories/%s/items",
```

Comment:
From my understanding this api call is meant for retrieving contents of files, isn't it (https://docs.microsoft.com/en-us/rest/api/azure/devops/git/items?view=azure-devops-rest-6.0)?
Do you think get commits api might be a better choice?
https://docs.microsoft.com/en-us/rest/api/azure/devops/git/commits/get-commits?view=azure-devops-rest-6.0
it can also receive `searchCriteria.$top` param that we could use.

I also noticed that we use `HEAD` in git implementation, but don't do it here. Is it a problem?

Reply:
The "scopePath=/" query param is applied while calling the "_apis/git/repositories/%s/items" API. So it returns only the root directory information but the contents of files. 
The "get-commits" is an option too. But considering the downloadZipFromAzureDevOps function uses the "items" API, I use the "items" API here too.

It's because the "refs" API of azure never returns the "HEAD" ref, I use "items" instead here. So it's not a problem.